src/data/scripts/hullmods/VayraDamagedShields.java

```diff
@@ -6,14 +6,29 @@
 import com.fs.starfarer.api.combat.ShipAPI.HullSize;
 import com.fs.starfarer.api.impl.campaign.ids.Stats;
 import com.fs.starfarer.api.impl.hullmods.CompromisedStructure;
+import data.scripts.util.MiscUtils;
+import data.util.LoggerLogLevel;
 
 public class VayraDamagedShields extends BaseHullMod {
+    private static final String LOGTAG = "VayraDamagedShields";
+    public static boolean DISABLE_FOR_PLAYER = false;
+    public static boolean DISABLE_FOR_ENEMY = false;
```

Comment:
also make these volatile so that the compiler never decides to inline them and that they always have to be evaluated properly

Reply:
fuuuck

Reply:
done everywhere. resolving.